tools/generator/JavaInteropCodeGenerator.cs

```diff
@@ -56,7 +56,7 @@ protected virtual string GetPeerMembersType ()
 
 		internal override void WriteClassHandle (InterfaceGen type, StreamWriter sw, string indent, CodeGenerationOptions opt, string declaringType)
 		{
-			sw.WriteLine ("{0}static JniPeerMembers _members = new JniPeerMembers (\"{1}\", typeof ({2}));",indent, type.RawJniName, declaringType);
+			sw.WriteLine ("{0}new static JniPeerMembers _members = new JniPeerMembers (\"{1}\", typeof ({2}));",indent, type.RawJniName, declaringType);
```

Comment:
I'm skeptical about this adding "new" is harmless, but not sure. It's rather @jonpryor who'd judge that as he's designed the overall internals.

Reply:
That `new` just removes many warnings in binding assemblies.

It'll instead introduce *different* warnings in `Mono.Android.dll` (particularly in `Java.Lang.Object`), but *semantically* this change is *not* a problem. It is in fact desirable.